manager/manager.go

```diff
@@ -71,7 +71,7 @@ type Config struct {
 
 	// RemoteAPI is a listening address for serving the remote API, and
 	// an optional advertise address.
-	RemoteAPI RemoteAddrs
+	RemoteAPI *RemoteAddrs
```

Comment:
I meant why not just check the default value, as opposed to nil?  If `ListenAddr` is `""`, there is no remote api set, right?  I'm not arguing that's better or anything - more just asking pointers are generally preferred over checking against the default value.